PokemonCard-v1.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+
+function PokemonCard() {
+
+ const [pokemonData, setPokemonData ] = useState({})
+ const [endpoint, setEndpoint] = useState('')
+
+   useEffect(() => {
+
+    async function fetchData() {
+
+      	try {
+          const res = await axios.get(`https://pokeapi.co/api/v2/pokemon/${ endpoint }`);
+          console.log(res.data);
+
+          setPokemonData(res.data);
+          
+        } catch(e) {
+
+          console.error(e)
+
+        }
+      
+    }
+   if(endpoint) {
+    fetchData();
+   }
+
+  }, [ endpoint ])
+
+
+  const handleJigglypuffClick = () => {
+    setEndpoint("jigglypuff");
+  };
+
+  const handleDittoClick = () => {
+    setEndpoint("ditto");
+  };
+  
```

Comment:
Als je hier
```suggestion
    if (!pokemonData) return null;
```
doet, dan kun je de check die hieronder staat weghalen en ben je wat nesting kwijt